### ISSUE TITLE
No output for classes with constructor

### ISSUE THREAD

**Issue by soulfresh** [jsdoc2md/jsdoc-to-markdown#240]
I'm probably missing something really obvious here but the following produces no output for me with the latest version of `jsdoc-to-markdown` when I run `npx jsdoc2md ./foo.js`

```js
/**
  * This is foo
  */
class Foo {
  constructor() {}
}
```

But this does produce output:

```js
/**
  * This is foo
  */
class Foo {
}
```

And this works too:

```js
class Bar {
  constructor() {}
}

/**
  * This is foo
  */
class Foo extends Bar {
}
```

Am I missing something obvious? I couldn't find any related bugs in this repo or in the `jsdoc` repo.

**Comment by 75lb** [comment]
In Javascript, a `Class` and a `constructor` are the same thing - the former is syntactic sugar for the latter.. If I remember correctly, you should only document either the `@class` or the `constructor`, but not both (as then you will have two sets of docs for the same JS constructor).. 

Generally, i think you should document the constructor where you can also document your `@param`s etc..

**Comment by soulfresh** [comment]
Totally but after playing with it more I've found the issue. I noticed that I forgot to include `export` statements in my examples which appear to be what breaks the docs.

For example, this produces no output:
```js
export class Foo {
  /**
   * This is foo
   */
  constructor() {}
}
```

But this does produce output:
```js
class Foo {
  /**
   * This is foo
   */
  constructor() {}
}

export { Foo };
```

So it appears that the export statement before the class declaration is what breaks the output when I run `npx jsdoc2md ./foo.js`. However, if I use `npx jsdoc2md ./foo.js --jsdoc` it does output the jsDoc data so maybe this is an issue with `jsdoc-to-markdown`? Either way, I at least have a work around now. Thanks for your quick response!
